pallets/liquidity-pools/src/lib.rs

```diff
@@ -888,14 +883,15 @@ pub mod pallet {
 				Error::<T>::AssetNotLiquidityPoolsTransferable
 			);
 
+			// TODO-1.7.2, should be v4?
```

Comment:
We need either:
- Support v3 and v4
- Migrate to asset registry to v4 and only use v4

Reply:
Current solution:
 - We convert any v3 xcm message to v4 because they are isometric, and we only treat in the code the v4 message.